src/cdk-experimental/listbox/listbox.ts

```diff
@@ -143,13 +202,44 @@ export class CdkListbox implements AfterContentInit, OnDestroy {
     this._disabled = coerceBooleanProperty(value);
   }
 
+  /** Whether the listbox will use active descendant or will move focus onto the options. */
+  @Input()
+  get useActiveDescendant(): boolean {
+    return this._useActiveDescendant;
+  }
+  set useActiveDescendant(shouldUseActiveDescendant: boolean) {
+    this._useActiveDescendant = shouldUseActiveDescendant;
+  }
+
+  ngOnInit() {
+    this._selectionModel = new SelectionModel<CdkOption>(this.multiple);
+  }
+
   ngAfterContentInit() {
     this._listKeyManager = new ActiveDescendantKeyManager(this._options)
       .withWrap().withVerticalOrientation().withTypeAhead();
+
+    this._listKeyManager.change.subscribe(() => {
+      this.updateActiveOption();
+    });
+
+    this._selectionModel.changed.pipe(takeUntil(this._destroy))
+        .subscribe((event: SelectionChange<CdkOption>) => {
+
+      event.added.forEach((option: CdkOption) => {
```

Comment:
Do you know if for/of loops should be used here? That's the Google internal style

Reply:
Yeah I've used them elsewhere, I forgot for this one. Thanks for the catch.